medcat/config_meta_cat.py

```diff
@@ -48,11 +47,19 @@ class Config:
 
 class Model(MixingConfig, BaseModel):
     """The model part of the metaCAT config"""
-    model_name: str = 'lstm'
+    model_name: str = 'bert'
+    """NOTE: When changing model, make sure to change the tokenizer as well"""
+    model_variant: str = 'bert-base-uncased'
+    model_freeze_layers: bool = True
     num_layers: int = 2
-    input_size: int = 300
+    input_size: int = 100
```

Comment:
Changing the default here could change existing behaviour. We want to avoid that.

If this is relevant to BERT but not LSTM, perhaps a separate method to get a default config suitable for BERT rather than LSTM?
If this should change in general, we can leave it be.

Reply:
I've reverted the change, input_size has to be 768 for bert, this is only for LSTM.